### PR TITLE
update action documentation with information about get-scenarios

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,16 +39,20 @@ The [check-controller-build](.github/actions/check-controller-build) action is u
 
 #### get-benchmarks
```
Comment (rafaelfolco) [inline README.md:40]: get-benchmarks (deprecated)

**Review by rafaelfolco — CHANGES_REQUESTED** [review]
could you just make it more clear ?

File: README.md (hunk 1 above)
Comment (rafaelfolco) [reply README.md:40]: just add a more clore note about deprecation at the top...
```
# foobar
Note: This option is **DEPRECATED**
```

Comment (k-rister) [reply README.md:40]: How is it now?

**Review by rafaelfolco — APPROVED** [review]

**Pull request merged**